sharding-spring/sharding-jdbc-orchestration-spring/pom.xml

```diff
@@ -33,12 +33,6 @@
     </modules>
     
     <dependencies>
-        <dependency>
-            <groupId>org.apache.shardingsphere</groupId>
-            <artifactId>sharding-jdbc-orchestration</artifactId>
-            <version>${project.version}</version>
-        </dependency>
-        
```

Comment:
Why move this dependence to it's children pom?
I think in parent is ok?

Reply:
This is an interim solution, I modify the children modules step by step. At last i will move this dependence to the parent pom.